crates/node/tests/identity.rs

```diff
@@ -0,0 +1,97 @@
+use calimero_node::{Node, NodeConfig};
+use calimero_primitives::context::ContextId;
+use camino::Utf8PathBuf;
+use libp2p::identity::Keypair;
+use tokio::sync::{broadcast, oneshot};
+use {calimero_context, calimero_network, calimero_server, calimero_store};
```

Comment:
this is unnecessary, deps are made available by default

and if they weren't, `use {<crate>};` is not a replacement for `extern crate <crate>;`

Reply:
I'll update tests once I know the source code is all correct 🙂